src/core/theme/utils/spacing.ts

```diff
@@ -1,19 +1,29 @@
-// TODO make this whole util again to work with suomifiTheme-util
-
+import { FlattenSimpleInterpolation } from 'styled-components';
 import {
-  spacingSuomifiTokens,
+  SpacingProp,
   spacingTokensKeys,
-  spacingTokens,
+  SpacingDesingTokens,
 } from '../spacing';
-import { SuomifiTokens } from '../';
+import { cssValueToString } from '../../../utils/css';
+
+export const spacingUtils = (spacing: SpacingDesingTokens) =>
+  Object.entries(spacing).reduce(
+    (retObj, [key, value]) => ({
+      ...retObj,
+      [key]: cssValueToString(value),
+    }),
+    {} as { [key in keyof SpacingDesingTokens]: FlattenSimpleInterpolation },
+  );
+
+// TODO make this whole util again to work with suomifiTheme-util
```

Comment:
Is this **TODO** comment still valid or is it already done and could be removed?